components/apimgt/org.wso2.carbon.apimgt.rest.api.devops/src/test/java/org/wso2/carbon/apimgt/rest/api/devops/RestApiDevopsUtilTest.java

```diff
@@ -54,4 +60,20 @@ public void testValidateLogLevelTestcase() throws JsonProcessingException {
         boolean logLevelSomething = DevopsAPIUtils.validateLogLevel("abc");
         Assert.assertEquals("Log level 'abc' should not be valid log level", false, logLevelSomething);
     }
+
+
+    @Test
+    public void testValidateCorrelationComponentList() throws APIManagementException {
```

Comment:
Shall we add some faulty scenarios as well to test the `DevopsAPIUtils.validateCorrelationComponentList`?

Reply:
Sure